src/diffusers/loaders/lora_base.py

```diff
@@ -305,10 +311,14 @@ def _fetch_state_dict(
                     user_agent=user_agent,
                 )
                 state_dict = load_state_dict(model_file)
+                file_extension = os.path.basename(model_file).split(".")[-1]
+                if file_extension == SAFETENSORS_FILE_EXTENSION:
+                    with safetensors.safe_open(model_file, framework="pt", device="cpu") as f:
+                        metadata = f.metadata()
```

Comment:
will we have any way or desire to warn users loading a lora from a pth file that we can't scale it?

Reply:
We currently error out when trying to save metadata to a pth file (or more generally, when `use_safetensors` is False).

Reply:
good! i never use them. but i was concerned for others.